googleapis/google-cloud-node
# migrate code from googleapis/nodejs-access-context-manager

- feat: initial stub of library
- feat: add templated files from docker image
- fix: nodejs package name access-context-manager (#4)
- chore(deps): update dependency mocha to v9 (#2)
- chore(deps): update dependency sinon to v11 (#3)
- build: correct repository field (#5)
- chore: release 0.2.0 (#7)
- docs: update reference docs url (#9)
- build(node): update deps used during postprocessing (#1243) (#8)
- chore: fix the wrong post processor image (#12)
- chore(deps): update dependency @types/node to v16 (#11)
- chore(deps): update dependency sinon to v12 (#13)
- docs(samples): add example tags to generated samples (#14)
- build: add generated samples to .eslintignore (#15)
- docs(node): support "stable"/"preview" release level (#1312) (#18)
- chore: add api_shortname and library_type to repo metadata (#17)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#20)
- test(nodejs): remove 15 add 16 (#1322) (#22)
- build: update copyright year to 2022 (#24)
- chore: update v2.12.0 gapic-generator-typescript (#26)
- chore(deps): update dependency sinon to v13 (#28)
- docs(nodejs): version support policy edits (#1346) (#30)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#31)
- chore: update v2.14.2 gapic-generator-typescript (#35)
- chore: remove unused imports (#36)
- feat: promote to stable (#37)
- chore(main): release 1.0.0 (#38)
- build(node): update client library version in samples metadata (#1356) (#45)
- chore(deps): update dependency sinon to v14 (#48)
- build!: update library to use Node 12 (#50)
- chore(main): release 2.0.0 (#51)
- chore(deps): update dependency jsdoc-region-tag to v2 (#54)
- chore(deps): update dependency jsdoc-fresh to v2 (#53)
- feat: support regapic LRO (#55)
- chore(main): release 2.1.0 (#56)
- chore(deps): update dependency linkinator to v4 (#60)
- fix: better support for fallback mode (#61)
- fix: change import long to require (#62)
- chore(main): release 2.1.1 (#63)
- fix: do not import the whole google-gax from proto JS (#1553) (#66)
- fix: allow passing gax instance to client constructor (#67)
- fix: preserve default values in x-goog-request-params header (#69)
- test: use fully qualified request type name in tests (#70)
- fix(deps): use google-gax v3.5.2 (#74)
- chore(deps): update dependency @types/node to v18 (#72)
- chore(deps): update dependency jsdoc to v4 (#76)
- fix: regenerated protos JS and TS definitions (#77)
- feat: added AccessPolicy.scopes, EgressTo.external_resources, and IAM methods (#78)
- chore(main): release 2.2.0 (#68)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
